extensions/wikia/Blogs/tests/BlogsHelperTest.php

```diff
@@ -8,8 +8,7 @@ public function setUp() {
 
 	/**
 	 * SUS-1635: Regression test for BlogsHelper::AbortMove hook handler
-	 * Verify blog pages and comments can be renamed, pages can be moved into blog namespace and that pages can't be moved
-	 * out of blogs namespaces or into comment namespace
+	 * Verify blog pages and comments can be renamed, pages can be moved into blog namespace and back.
```

Comment:
Confusing comment :) The code that is tested says that we want to prevent moving pages **into** blog namespace.